CLAUDE.md

```diff
@@ -0,0 +1,92 @@
+# Claude Code Memory for Notecard Schema Repository
+
+## Repository Overview
+This repository contains JSON schemas for the Notecard API, following JSON Schema Draft 2020-12 specification. The schemas are used to generate the [Notecard API Reference Documentation](https://dev.blues.io/api-reference/).
+
+## Schema Creation Process
+When adding a new schema (e.g., `card.sleep`):
+
+1. **Create schema files**: Both `.req.notecard.api.json` and `.rsp.notecard.api.json` files
+2. **Add comprehensive tests**: Create corresponding test files in `tests/` directory
+3. **Update index**: Add schema references to `notecard.api.json`
+4. **Verify descriptions**: Ensure all descriptions match exactly with official API documentation
```

Comment:
It may be worth noting that, "Any markdown compatible formatting should be preserved."